diff -rauB a/src/firejail/Makefile.in b/src/firejail/Makefile.in
--- a/src/firejail/Makefile.in	2015-03-12 02:34:30.000000000 +0200
+++ b/src/firejail/Makefile.in	2015-06-21 03:06:58.657436138 +0300
@@ -12,7 +12,7 @@
 C_FILE_LIST       = $(wildcard *.c)
 OBJS = $(C_FILE_LIST:.c=.o)
 BINOBJS =  $(foreach file, $(OBJS), $file)
-CFLAGS += -ggdb -O2 -DVERSION='"$(VERSION)"' -DPREFIX='"$(PREFIX)"' $(HAVE_SECCOMP) $(HAVE_SECCOMP_H) $(HAVE_CHROOT) $(HAVE_BIND) -fstack-protector-all -D_FORTIFY_SOURCE=2 -fPIE -pie -Wformat -Wformat-security
+CFLAGS += -ggdb -O2 -DVERSION='"$(VERSION)"' -DPREFIX='"$(PREFIX)"' $(HAVE_SECCOMP) $(HAVE_SECCOMP_H) $(HAVE_CHROOT) $(HAVE_BIND) -fstack-protector-all -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=2 -fPIE -pie -Wformat -Wformat-security
 LDFLAGS:=-pie -Wl,-z,relro -Wl,-z,now -lpthread
 
 %.o : %.c $(H_FILE_LIST)
diff -rauB a/src/firemon/Makefile.in b/src/firemon/Makefile.in
--- a/src/firemon/Makefile.in	2015-03-12 02:34:58.000000000 +0200
+++ b/src/firemon/Makefile.in	2015-06-21 03:07:24.350434619 +0300
@@ -8,7 +8,7 @@
 C_FILE_LIST       = $(wildcard *.c)
 OBJS = $(C_FILE_LIST:.c=.o)
 BINOBJS =  $(foreach file, $(OBJS), $file)
-CFLAGS += -ggdb -O2 -DVERSION='"$(VERSION)"' -fstack-protector-all -D_FORTIFY_SOURCE=2 -fPIE -pie -Wformat -Wformat-security
+CFLAGS += -ggdb -O2 -DVERSION='"$(VERSION)"' -fstack-protector-all -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=2 -fPIE -pie -Wformat -Wformat-security
 LDFLAGS:=-pie -Wl,-z,relro -Wl,-z,now 
 
 %.o : %.c $(H_FILE_LIST)
diff -rauB a/src/ftee/Makefile.in b/src/ftee/Makefile.in
--- a/src/ftee/Makefile.in	2015-03-12 02:34:09.000000000 +0200
+++ b/src/ftee/Makefile.in	2015-06-21 03:07:53.744432882 +0300
@@ -8,7 +8,7 @@
 C_FILE_LIST       = $(wildcard *.c)
 OBJS = $(C_FILE_LIST:.c=.o)
 BINOBJS =  $(foreach file, $(OBJS), $file)
-CFLAGS += -ggdb -O2 -DVERSION='"$(VERSION)"' -DPREFIX='"$(PREFIX)"' -fstack-protector-all -D_FORTIFY_SOURCE=2 -fPIE -pie -Wformat -Wformat-security
+CFLAGS += -ggdb -O2 -DVERSION='"$(VERSION)"' -DPREFIX='"$(PREFIX)"' -fstack-protector-all -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=2 -fPIE -pie -Wformat -Wformat-security
 LDFLAGS:=-pie -Wl,-z,relro -Wl,-z,now -lpthread
 
 %.o : %.c $(H_FILE_LIST)
diff -rauB a/src/lib/Makefile.in b/src/lib/Makefile.in
--- a/src/lib/Makefile.in	2014-07-11 21:09:38.000000000 +0300
+++ b/src/lib/Makefile.in	2015-06-21 03:18:32.635395110 +0300
@@ -6,7 +6,7 @@
 C_FILE_LIST       = $(wildcard *.c)
 OBJS = $(C_FILE_LIST:.c=.o)
 BINOBJS =  $(foreach file, $(OBJS), $file)
-CFLAGS += -ggdb -O2 -DVERSION='"$(VERSION)"' -fstack-protector-all -D_FORTIFY_SOURCE=2 -fPIC -Wformat -Wformat-security
+CFLAGS += -ggdb -O2 -DVERSION='"$(VERSION)"' -fstack-protector-all -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=2 -fPIC -Wformat -Wformat-security
 LDFLAGS:=-pic -Wl,-z,relro -Wl,-z,now 
 
 all: $(OBJS)
diff -rauB a/src/libtrace/Makefile.in b/src/libtrace/Makefile.in
--- a/src/libtrace/Makefile.in	2014-10-07 18:32:00.000000000 +0300
+++ b/src/libtrace/Makefile.in	2015-06-21 03:09:19.214427829 +0300
@@ -6,7 +6,7 @@
 C_FILE_LIST       = $(wildcard *.c)
 OBJS = $(C_FILE_LIST:.c=.o)
 BINOBJS =  $(foreach file, $(OBJS), $file)
-CFLAGS += -ggdb -O2 -DVERSION='"$(VERSION)"' -fstack-protector-all -D_FORTIFY_SOURCE=2 -fPIC -Wformat -Wformat-security
+CFLAGS += -ggdb -O2 -DVERSION='"$(VERSION)"' -fstack-protector-all -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=2 -fPIC -Wformat -Wformat-security
 LDFLAGS:=-pic -Wl,-z,relro -Wl,-z,now 
 
 all: libtrace.so
